static/css/page-plain.less

```diff
@@ -24,7 +24,6 @@ body#plain {
   /* stylelint-disable selector-max-specificity */
   #contentBody,
   #contentHead {
-    width: 690px;
```

Comment:
I like it, but are you  sure this doesn't break anything?

Reply:
Yes, tested